prov/tcp/src/tcpx_progress.c

```diff
@@ -144,6 +145,47 @@ static int tcpx_process_remote_write(struct tcpx_pe_entry *pe_entry)
 	return FI_SUCCESS;
 }
 
+static int tcpx_process_remote_read_req(struct tcpx_pe_entry *pe_entry)
+{
+	struct tcpx_ep *tcpx_ep = pe_entry->ep;
+	struct tcpx_cq *tcpx_cq;
+	struct tcpx_pe_entry *resp_entry;
+	int i;
+
+	tcpx_cq = container_of(tcpx_ep->util_ep.rx_cq, struct tcpx_cq,
+			       util_cq);
+
+	resp_entry = tcpx_pe_entry_alloc(tcpx_cq);
+	if (!resp_entry)
+		return -FI_EAGAIN;
```

Comment:
This request came from the peer.  Where does this EAGAIN end up going to and what is done with it?